Inspicio/wwwroot/js/ajaxComments.js

```diff
@@ -173,12 +173,21 @@ function commentClick(uniqID, chosenState) {
 
     //Get new comments for markers
     function reloadMarkers() {
-        
+        var controllers = ["Images", "Account", "Images", "Manage"];
+        var path = location.pathname.split("/");
+        var method;

```

Comment:
This code seems to be duplicated from elsewhere, if you do need to keep it duplicate it's a good idea to tag with a comment so that future-you knows to update all copies of it if something changes (I usually put a [guid](https://www.guidgenerator.com/) in the comment so it's easy to search for all other places with that guid), or pull this code out into its own function that both places use.